domain/RealtorUpsertProperty.go

```diff
@@ -92,6 +92,7 @@ func (d *Domain) RealtorUpsertProperty(in *RealtorUpsertPropertyIn) (out Realtor
 		prop.SetCreatedBy(sess.UserId)
 	}
 	prop.SetUniqPropKey(fmt.Sprintf(`%d_%d`, sess.UserId, fnv1a.HashString64(in.Property.FormattedAddress)))
+	fmt.Println("Altitude = ", in.Property.Altitude)
```

Comment:
can just send from client side with `debug: 1` or `?debug=1` to print whetever server receive and out.*